src/main/java/org/akhq/models/Record.java

```diff
@@ -71,12 +72,16 @@
     private byte[] bytesValue;
 
     @Getter(AccessLevel.NONE)
+    @Setter(AccessLevel.NONE)
     private String value;
 
     private final List<String> exceptions = new ArrayList<>();
 
     private byte MAGIC_BYTE;
 
+    @Setter(AccessLevel.NONE)
```

Comment:
You need to remove this one, other else the value will never be sent on the ui part